_site/index.html

```diff
@@ -0,0 +1,205 @@
+</span></pre>
+<hr />
+<p>If you are running Windows, download and run the <a href="https://github.com/bytecodealliance/wasmtime/releases/download/dev/wasmtime-dev-x86_64-windows.msi">Wasmtime
+Installer</a>
+then follow the on-screen instructions.</p>
+<hr />
+<p>Alternatively, you can download releases for all supported platforms from the
+<a href="https://github.com/bytecodealliance/wasmtime/releases">Wasmtime GitHub page</a>.</p>
+</div>
+</div>
+</section>
+<section class="section-tinted">
+<div class="container w-container">
+<h2>Example</h2>
+<p>If you've got the <a href="https://www.rust-lang.org/tools/install">Rust compiler
+installed</a> then you can take some Rust
+source code:</p>
+<pre style="background-color:#2b303b;">
+<span style="color:#b48ead;">fn </span><span style="color:#8fa1b3;">main</span><span style="color:#c0c5ce;">() {
+</span><span style="color:#c0c5ce;">    println!(&quot;</span><span style="color:#a3be8c;">Hello, world!</span><span style="color:#c0c5ce;">&quot;);
+</span><span style="color:#c0c5ce;">}
+</span></pre>
+<p>and compile/run it with:</p>
+<pre style="background-color:#2b303b;">
+<span style="color:#8fa1b3;">$</span><span style="color:#c0c5ce;"> rustup target add wasm32-wasi
+</span><span style="color:#8fa1b3;">$</span><span style="color:#c0c5ce;"> rustc hello.rs</span><span style="color:#bf616a;"> --target</span><span style="color:#c0c5ce;"> wasm32-wasi
+</span><span style="color:#8fa1b3;">$</span><span style="color:#c0c5ce;"> wasmtime hello.wasm
+</span><span style="color:#8fa1b3;">Hello,</span><span style="color:#c0c5ce;"> world!
+</span></pre>
+</div>
+</section>
+<section>
+<div class="container w-container">
+<h2>Features</h2>
+<ul>
+<li>
+<p><strong>Fast</strong>. Wasmtime is built on the optimizing <a href="https://github.com/bytecodealliance/wasmtime/blob/main/cranelift/README.md">Cranelift</a> code generator to
+quickly generate high-quality machine code either at runtime or
+ahead-of-time. Wasmtime's runtime is also optimized for cases such as
+efficient instantiation, low-overhead transitions between the embedder and
+wasm, and scalability of concurrent instances.</p>
+</li>
+<li>
+<p><strong><a href="https://docs.wasmtime.dev/security.html">Secure</a></strong>. Wasmtime's development is strongly focused on the correctness of
+its implementation with 24/7 fuzzing donated by <a href="https://google.github.io/oss-fuzz/">Google's OSS Fuzz</a>,
+leveraging Rust's API and runtime safety guarantees, careful design of
+features and APIs through an <a href="https://github.com/bytecodealliance/rfcs">RFC process</a>, a <a href="https://bytecodealliance.org/security">security policy</a> in place
+for when things go wrong, and a <a href="https://docs.wasmtime.dev/stability-release.html">release policy</a> for patching older versions
+as well. We follow best practices for defense-in-depth and known
+protections and mitigations for issues like Spectre. Finally, we're working
```

Comment:
I'm finding this first sentence pretty tough to read. I would re-work it or break it up a bit. Maybe something like:

```suggestion
<p><strong><a href="https://docs.wasmtime.dev/security.html">Secure</a></strong>. Wasmtime's development is strongly focused on correctness and security. Building on top of Rust's runtime safety guarantees, each Wasmtime feature undergoes careful review and consideration via an <a href="https://github.com/bytecodealliance/rfcs">RFC process</a>. Once features are designed and implemented, they undergo 24/7 fuzzing donated by <a href="https://google.github.io/oss-fuzz/">Google's OSS Fuzz</a>. As features stabilize they become part of a <a href="https://docs.wasmtime.dev/stability-release.html">release</a>, and when things go wrong we have a well-defined <a href="https://bytecodealliance.org/security">security policy</a> in place to quickly mitigate and patch any issues. We follow best practices for defense-in-depth and integrate protections and mitigations for issues like Spectre. Finally, we're working
```